### ISSUE TITLE
Fix Code Folding tests

### ISSUE THREAD

**Issue by petetnt** [adobe/brackets#12724]
There are a few failing tests now in the latest master:

> can be enabled by setting `makeSelectionsFoldable' to true

``` javascript
Error: Expected [ 0, 10, 16, 20, 24, 26, 29 ] to contain 2.
    at new jasmine.ExpectationResult (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:114:32)
    at null.toContain (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:1235:29)
    at null.<anonymous> (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/extensions/default/CodeFolding/unittests.js:430:47)
    at jasmine.Block.execute (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:1064:17)
    at jasmine.Queue.next_ (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2096:31)
    at onComplete (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2092:18)
    at jasmine.WaitsForBlock.execute (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2576:5)
    at file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2590:12
```
> shows fold ranges for only the most recent selection

``` js
Error: Expected [ 0, 10, 16, 20, 24, 26, 29 ] to contain 5.
    at new jasmine.ExpectationResult (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:114:32)
    at null.toContain (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:1235:29)
    at null.<anonymous> (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/extensions/default/CodeFolding/unittests.js:459:47)
    at jasmine.Block.execute (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:1064:17)
    at jasmine.Queue.next_ (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2096:31)
    at file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2086:18
```

> can be enabled by setting `makeSelectionsFoldable' to true
``` javascript
Error: Expected [ 0, 1, 4, 6, 7, 11, 12, 13, 17, 18, 23, 26 ] to contain 3.
    at new jasmine.ExpectationResult (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:114:32)
    at null.toContain (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:1235:29)
    at null.<anonymous> (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/extensions/default/CodeFolding/unittests.js:430:47)
    at jasmine.Block.execute (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:1064:17)
    at jasmine.Queue.next_ (file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2096:31)
    at file:///C:/Program%20Files%20(x86)/Brackets/dev/test/thirdparty/jasmine-core/jasmine.js:2086:18
```

cc @thehogfather

**Comment by thehogfather** [comment]
@petetnt I'm actually unable to verify that tests are now passing as I just get a blank white screen on the tests window. Is this a known issue?

**Comment by designcise** [comment]
I'm using v1.8 build 1.8.0-17080; code folding is still breaking in this version. Also it would be nice to have code folding for switch's cases, thanks!

**Comment by ficristo** [comment]
@thehogfather do you still have the problem? if so, have you tryed to run `npm install`?

@designcise do you have clear step to reproduce? If so can you open a new issue?

**Comment by ficristo** [comment]
@designcise for the `case` you could try to add `code-folding​.alwaysUseIndentFold: true` to your preferences.

**Comment by thehogfather** [comment]
@ficristo thanks - `npm install` seem to have addressed the issue. Code folding tests off the master branch are passing.
Also this issue can be safely closed as it was introduced by 5727863 which has since been reverted.
